website/docs/Getting Started.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 1
+---
+
+A Rust framework for building Minecraft: Java Edition servers.
+
+Like [feather](https://github.com/feather-rs/feather), Valence is an effort to build a Minecraft compatible server
+completely from scratch in Rust. The difference is that Valence has decided to organize the effort a little differently.
+All game logic is behind a trait. This approach has many advantages. Features such as a plugin system, dedicated
+executable, and vanilla game mechanics can be implemented _on top of_ Valence. Valence is a Rust library like any other.
+
+In the future we may decide to reimplement vanilla game mechanics as a separate project. If you're developing something
+like a minigame server without need for vanilla game mechanics, you can depend on Valence directly.
+
+# Goals
+
+Valence aims to be the following:
+
+* **Complete**. Abstractions for the full breadth of the Minecraft protocol.
+* **Flexible**. Your use case should be achievable without manually sending and receiving packets or other hacks.
+* **Minimal**. The API surface is small with only the necessities exposed. Opinionated features such as a
```

Comment:
From here the description about unopinionatedness can be moved into a separated goal like `Unopinionated`.

Reply:
Sure, this wasn't too much focused on the contents, more on the system. Feel free to send PRs to the fork, if you have some exact ideas. Thanks.